README.md

```diff
@@ -27,6 +31,10 @@ In order to run GraphSpace, please install postgreSQL and both the Python runtim
 3. Create a virtual environment for the project: `virtualenv venv`
 4. Start using the virtual environment: `source venv/bin/activate`
 5. Install graphspace: `sh install.sh`
+   
+   _Note : While installing psycopg2, you might encounter 
+   [```Error: You need to install postgresql-server-dev-X.Y for building a server-side extension or libpq-dev for building a client-side application.```](https://stackoverflow.com/q/28253681/4646197)
+   To fix it follow the solution given [here](https://stackoverflow.com/a/28254860/4646197)._
```

Comment:
I feel its outside the scope of the README to mention about an error that might happen. More like 90% of the people won't get this error so didn't want to disturb the flow of steps by adding commands.